pandas/tests/frame/test_api.py

```diff
@@ -337,12 +337,10 @@ def test_constructor_expanddim_lookup(self):
         df = DataFrame()
 
         with warnings.catch_warnings(record=True) as wrn:
-            # _AXIS_NUMBERS, _AXIS_NAMES lookups
             inspect.getmembers(df)
 
-        # some versions give FutureWarning, others DeprecationWarning
-        assert len(wrn)
-        assert any(x.category in [FutureWarning, DeprecationWarning] for x in wrn)
+        # some versions may give FutureWarning, others DeprecationWarning
+        assert not len(wrn)
```

Comment:
This seems like unrelated to the actual test for constructor_expanddim. Maybe split it in two separate tests?